PrioritiesSprint5.md

```diff
@@ -65,6 +85,9 @@ between July 2 and July 9, 2024
 
 # CLEAN-UP EFFORT: Documentation 
 ## Pre-development
+- Each team closes Guppy documentation issues
 
 # Publications
-## Pre-development
\ No newline at end of file
+## Pre-development
+- Circulate Outline of proposed pub
+- Collect feedback and updates of outline
```

Comment:
Goal is to collect feedback and use cases during Sprint #5 . The outline will be updated and v1 will be prepared in Sprint #6.